docs/modules/ROOT/pages/release-notes/release-notes-28-1.adoc

```diff
@@ -0,0 +1,27 @@
+= Payara Enterprise 5.28.1 Release Notes
+
+== Supported APIs and Applications
+
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 3.3
+
+== Notes
+
+=== Incompatibility with the latest JVM / JDK versions and Monitoring
+IMPORTANT: Payara Platform products do run correctly on JDK 8u292+ and 11.0.11+ due to the removal of SSL/TLS  1.0 and SSL/TLS 1.1 protocols in these versions. The JMX service doesn't start properly which means there are no monitoring data available.
```

Comment:
This paragraph is very confusing to me.

If I understand it correctly that Payara Platform runs OK on the latest JDK versions and only JMS service doesn't start, then it should clearly state that:

* Due to removal of TLS 1.0 and 1.1, monitoring data will not be available because the JMX service will fail to start and will not produce monitoring data.
* All the other functionality of Payara Platform except monitoring works correctly on the latest JDKs and Payara Platform can run on them in production without any issues

Reply:
Since we can't test on the latest JDK (the TCK and other tests fail) we cannot say that "other functionality" works correctly here